File: java_projects/glue_code/commons-fileupload/src/main/java/org/apache/commons/fileupload/util/mime/ParseException.java
package org.apache.commons.fileupload.util.mime;

import org.apache.commons.fileupload.ContextInitializer;
import org.graalvm.polyglot.Value;

final class ParseException extends Exception {
  private static final long serialVersionUID = 5355281266579392077L;
  private static Value clz =
      ContextInitializer.getPythonClass("/util/mimParseException.py", "ParseException");
  private Value obj;

  public ParseException(Value obj) {
    this.obj = obj;
  }

  public Value getPythonObject() {
    return obj;
  }

  ParseException(String message) {
    //
    // super(message);
    //

    this.obj = clz.invokeMember("__init__", message);
  }
}

File: data/recomposed_projects/commons-fileupload/src/main/org/apache/commons/fileupload/util/mime/ParseException.py
# Imports Begin
from src.main.org.apache.commons.fileupload.java_handler import java_handler
# Imports End


@java_handler
class ParseException(Exception):

    # Class Fields Begin
    __serialVersionUID: int = 5355281266579392077
    # Class Fields End

    # Class Methods Begin
    def __init__(self, message: str) -> None:

        super().__init__(message)

    # Class Methods End
